components/script/dom/node.rs

```diff
@@ -934,6 +934,18 @@ impl Node {
         self.AppendChild(&node).map(|_| ())
     }
 
+    // https://dom.spec.whatwg.org/#dom-parentnode-append
```

Comment:
```suggestion
    // https://dom.spec.whatwg.org/#dom-parentnode-replacechildren
```

Found wrong link though 👀

Reply:
My bad :-/
Fixed and rebased.

Reply:
Thank you so much ❤️